apps/institution-service/cypress/e2e/institutions.cy.ts

```diff
@@ -496,6 +496,63 @@ describe("/institutions", () => {
     );
   });
 
+  it("gets sorted institution list with default sort options", () => {
+    cy.request({
+      url: `http://localhost:${PORT}/institutions?aggregatorName=testExampleA`,
+      method: "GET",
+      headers: {
+        Authorization: createAuthorizationHeader(SUPER_USER_ACCESS_TOKEN_ENV),
+      },
+    }).then((response: Cypress.Response<{ body: unknown }>) => {
+      const institutionResponse =
+        response.body as unknown as PaginatedInstitutionsResponse;
+
+      expect(response.status).to.eq(200);
+      expect(institutionResponse.totalRecords).to.eq(3);
+      expect(institutionResponse.institutions[0].id).to.eq(
```

Comment:
This is quite brittle. If someone creates another testExampleA bank, then this test could fail.

You could check the results of a request and make sure that the values are ordered correctly